test/ctr.bats

```diff
@@ -1176,6 +1176,39 @@ function teardown() {
 	stop_crio
 }
 
+@test "ctr with run_as_user set to 100 should 101 as the gid for redis:alpine" {
```

Comment:
nit: should use 101

Reply:
the uid for redis is 100, but gid is 101.

Reply:
I meant the text: "ctr with run_as_user set to 100 should use 101 as the gid .." :)

Reply:
oh sorry, yeah fixed that :)